Problem001/Go/solution_1.go

```diff
@@ -13,5 +13,6 @@ func main() {
 			sum += i
 		}
 	}
+
```

Comment:
Why this?

Reply:
I used to modify solutions to proc them in `./test sync staged` so I have visibility of how stuff happens